src/components/theme/styles.js

```diff
@@ -20,7 +20,8 @@ export const theme = {
   fonts: {
     ...DefaultTheme.fonts,
     slab: 'RobotoSlab-Regular',
-    slabBold: 'RobotoSlab-Bold'
+    slabBold: 'RobotoSlab-Bold',
+    bold: 'Roboto-Bold'
```

Comment:
Just curious, this will render different than selecting 'Roboto' as the font and setting the `fontWeight` to `bold`?

If there's no difference, probably `slabBold` is not necessary either.

Reply:
I think they are different fonts, but not really sure. Maybe @gabitoesmiapodo can answer that question?

Reply:
Yeah, they are and look different, so if this typography is used in the design we should use that.